cmd/upgrade.go

```diff
@@ -164,13 +165,18 @@ func newChartCommand() *cobra.Command {
 						" --dry-run=server enables the cluster access with helm-get and the lookup template function."
 				)
 
+				cmdFlags := cmd.Flags()
+
+				// see: https://github.com/databus23/helm-diff/issues/537
+				cmdFlags.ParseErrorsWhitelist.UnknownFlags = unknownFlags
```

Comment:
Wow, good to know! Thanks..

So does this basically mean `DisableFlagParsing: true,` results in `UnknownFlags NOT set by cobra? (Just confirming for learning purpose

Reply:
Yes, this will disrupt a lot of functionality. I think we only support -- dry run=something. @mumoshu WDYT?

Reply:
@yxxhero Thanks! Re dry-run, it won't be a good idea to start dropping the backward-compatibility without any prenotices. We are still in the same major ver of helm-diff (3.x). Dropping the compatibility may break downstream user CI/CD pipelines and/or tools.

I think this justifies having more test coverage in future though 🤔

Reply:
@mumoshu yeah. make sense.